rollup-plugins/production-pruning.js

```diff
@@ -63,24 +63,28 @@ const formatMessage = (node) => {
 
 const createErrorDocumentation = (node) => {
   const errorMessage = formatMessage(node);
-  const errorIndex = errors.push(errorMessage);
+  let errorIndex = errors.indexOf(errorMessage);
 
-  const markdownOutput = `
-## Error ${errorIndex}:
+  if (errorIndex > -1) {
+    return errorIndex;
+  } else {
+    const errorIndex = errors.push(errorMessage);
 
+    const markdownOutput = `
+## Error Code ${errorIndex}:
```

Comment:
Don’t forget to update the code in the error link function to now use Error-Code vs Error

Reply:
I think Error is actually better, so I adjusted Error Code to Error in the markdown output

Reply:
Let me know what you think about that - it sure does shorten our URLs